QModPluginEmulator/QModPluginGenerator.cs

```diff
@@ -108,6 +108,13 @@ private static IEnumerator InitializeQMM(IEnumerator result)
 
                 SummaryLogger.ReportIssues(ModsToLoad);
                 SummaryLogger.LogSummaries(ModsToLoad);
+                if(Patcher.Dialogs.Count > 0)
```

Comment:
No need to check `Count` when using a `foreach`

Reply:
no clue why I did that lol 
bb148d0